.github/workflows/prow-action.yml

```diff
@@ -9,12 +9,21 @@ jobs:
   execute:
     runs-on: ubuntu-latest
     steps:
-      - uses: jpmcb/prow-github-actions@v1.1.3
+      - uses: jpmcb/prow-github-actions@v1
```

Comment:
Sorry for the misleading, the action repo doesn't have a v1 tag.
We should still use v1.1.3